src/__type_tests__/button-type-test.tsx

```diff
@@ -42,7 +37,7 @@ import {ButtonPrimary} from '..';
 
 <ButtonPrimary to="/foo">Text</ButtonPrimary>;
 
-<ButtonPrimary to="/foo" fullPageOnWebView>
+<ButtonPrimary to="/foo" newTab>
     Text
 </ButtonPrimary>;
```

Comment:
can you explain this? I don't find a deprecation message for `fullPageOnWebView`

Reply:
Yes, I think it's still needed, and used in webapp

Reply:
Restored